packages/office-addin-debugging/src/commands.ts

```diff
@@ -23,14 +38,13 @@ function parseDevServerPort(optionValue: any): number | undefined {
     return devServerPort;
 }
 
-export async function start(manifestPath: string, appType: string | undefined, command: commander.Command) {
+export async function start(manifestPath: string, platform: string | undefined, command: commander.Command) {
     try {
-        const appTypeToDebug: AppType | undefined = parseAppType(appType || process.env.npm_package_config_app_type_to_debug || "desktop");
+        const appPlatformToDebug: AppPlatform | undefined = parseAppPlatform(platform || process.env.npm_package_config_app_platform_to_debug || AppPlatform.Win32);
+        const appTypeToDebug: AppType | undefined = parseAppType(appPlatformToDebug || process.env.npm_package_config_app_type_to_debug || AppType.Desktop);
         const appToDebug: string | undefined = command.app || process.env.npm_package_config_app_to_debug;
         const app: OfficeApp | undefined = appToDebug ? parseOfficeApp(appToDebug) : undefined;
-        const sourceBundleUrlComponents = new devSettings.SourceBundleUrlComponents(
-            command.sourceBundleUrlHost, command.sourceBundleUrlPort,
-            command.sourceBundleUrlPath, command.sourceBundleUrlExtension);
+        const devBuild: boolean = command.prod ? false : true;
```

Comment:
Can this just be `const dev: boolean`? It would be consistent with other places.

I keep wondering about using something more like isDevMode or using an enum such as BuildFlavor or Mode = { Development, Production } but it does seem simpler to use a boolean. I think "dev" is OK if used consistently.

The replacement parameter is ${BuildFlavor} but I wonder if ${Mode} is better. Of course we could use "--mode production" which is more like webpack, but that is probably overkill. Anyway these various thoughts keep going through my head, but the simplest "dev: boolean" and "--prod" seems OK. I'm curious if you have any opinions.